challenge_set_a_table/README.md

```diff
@@ -2,9 +2,14 @@
 
 ## Testing
 
+Setup
+* Put the robot in front of the door (the usual initial pose)
+* Make sure one of the combos is on the cabinet (look at the RoboCup knowledge for the possibilities). N.B., only the 'easy' items are relevant.
+* When the robot asks what to do, answer 'Set the table'. The rest should speak for itself.
+
 Launch procedure:
 ```
-amigo-start
+amigo-start (usually already running)
 amigo-free-mode
 rosrun challenge_set_a_table challenge_set_a_table.py amigo
```

Comment:
Why not create a launch file?

Reply:
Updated in 90e90818e4d3ab3b0d62c0e93b6e6fe88aa18d45